### PR TITLE
Dublicate templates

### PR DESCRIPTION
**Pull request by danielfriis**

### CODE REVIEW

File: spec/features/template_dublications_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe TemplateDublicationsController do
+  let(:account) { create :account }
+  let(:user) { create :administrator, account: account }
+  let(:template) { create :template, account: account }
+  let(:template_field) { create :template_field, template: template }
+  let(:template_field) { create :template_field, template: template }
+
+  it "dublicates the template" do
```
Comment (kaspernj) [inline spec/features/template_dublications_spec.rb]: I always add something like this, to tell other developers which specific action I intend to test:

```ruby
describe "#create" do
```

File: spec/features/template_dublications_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe TemplateDublicationsController do
+  let(:account) { create :account }
+  let(:user) { create :administrator, account: account }
+  let(:template) { create :template, account: account }
+  let(:template_field) { create :template_field, template: template }
+  let(:template_field) { create :template_field, template: template }
```
Comment (kaspernj) [inline spec/features/template_dublications_spec.rb]: Duplicate lets?

Comment (danielfriis) [inline spec/features/template_dublications_spec.rb]: 👍

File: spec/features/template_dublications_spec.rb (hunk 1 above)
Comment (danielfriis) [inline spec/features/template_dublications_spec.rb]: 👍 fixer

**Pull request merged**